lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -863,6 +867,36 @@ void MtohRenderOverride::ClearHydraResources()
     _initializationAttempted = false;
 }
 
+void MtohRenderOverride::_CreateSceneIndicesChainAfterMergingSceneIndex()
+{
+    //This function is where happens the ordering of filtering scene indices that are after the merging scene index
+
+    HdSceneIndexBaseRefPtr lastSceneIndexOfTheChain = _renderIndexProxy->GetMergingSceneIndex();
+
+    _selection = std::make_shared<Fvp::Selection>();
+    _selectionSceneIndex = Fvp::SelectionSceneIndex::New(lastSceneIndexOfTheChain, _selection);
+    _selectionSceneIndex->SetDisplayName("Flow Viewport Selection Scene Index");
+    lastSceneIndexOfTheChain = _selectionSceneIndex;
+  
+    if (!_sceneIndexRegistry) {
+        _sceneIndexRegistry.reset(new MayaHydraSceneIndexRegistry(*_renderIndexProxy));
```

Comment:
@ppt-adsk : is that expected that _sceneIndexRegistry is not added to the render index proxy or isn't used as an entry of another scene index, it doesn't seem to be part of the chain ?

Reply:
@lanierd-adsk why should the scene index registry be part of the chain?  It is not a scene index.